update_alternatives_clang.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# update_alternatives_clang.sh
+# chmod u+x update_alternatives_clang.sh
+#
+
+update_alternatives() {
+    local version=${1}
+    local priority=${2}
+    local master=${3}
+    local slaves=${4}
```

Comment:
And so on..
```suggestion
    local leader=${3}
    local follower=${4}
```

Reply:
while I don't agree with the terminology, it is very present in the man pages for update-alternatives: https://man7.org/linux/man-pages/man1/update-alternatives.1.html

Reply:
Ok. But please move this script to `.devcontainer` folder. Or perhaps delete it altogether, if you don't plan to configure clang as default compiler.